src/components/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { View, TouchableOpacity } from 'react-native';
+import styles from './Button.styles';
+
+const Button = (props) => {
+  const { onClickAction, id, children } = props;
+  const { container } = styles;
+  return (
+    <View style={container}>
+      <TouchableOpacity onPress={(e) => { onClickAction(e, id); }}>
+        {children}
+      </TouchableOpacity>
+    </View>
+  );
+};
+
+export default Button;
```

Comment:
Right now this component is not being reused, but in future projects you could consider adding it in a *Common* directory

Reply:
If you are thinking of reusing, maybe you can pass the style as a prop too. Is that possible?

Reply:
I'll keep it in mind in the future for reusable components